src/channel.coffee

```diff
@@ -86,6 +86,19 @@ class Channel
     m = new Message @_client, {text: text}
     @sendMessage m
 
+  postMessage: (data) ->
+    params = data
+    params.channel = @id
+    if data.attachments
+      params.attachments = JSON.stringify(data.attachments)
+
+    @_client.logger.debug data
+    @_client.logger.debug params
+    @_client._apiCall "chat.postMessage", params, @_onPostMessage
+
+  _onPostMessage: (data) =>
```

Comment:
A fair point - I was just copying the existing pattern. I'm open to other approaches here!